spec/models/tslr_claim_spec.rb

```diff
@@ -285,75 +267,19 @@
   end
 
   describe "#ineligible?" do
-    subject { build(:tslr_claim, claim_attributes).ineligible? }
-
-    context "with an ineligible QTS award year" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, qts_award_year: "before_2013")} }
-      it { is_expected.to be true }
-    end
-
-    context "with an eligible QTS award year" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, qts_award_year: "2013_2014")} }
-      it { is_expected.to be false }
-    end
-
-    context "with no claim_school" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, claim_school: nil)} }
-      it { is_expected.to be false }
-    end
-
-    context "with an eligible claim school" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, claim_school: schools(:penistone_grammar_school))} }
-      it { is_expected.to be false }
-    end
-
-    context "with an ineligible claim_school" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, claim_school: schools(:hampstead_school))} }
-      it { is_expected.to be true }
-    end
-
-    context "when no longer teaching" do
-      let(:claim_attributes) { {eligibility: build(:student_loans_eligibility, employment_status: :no_school)} }
-      it { is_expected.to be true }
-    end
+    it "delegates to the eligibility" do
```

Comment:
We're not actually testing that it delegates here.

Reply:
Good shout on these, it felt wrong and I've now just removed the delegation altogether.